Source/OIDLogoutRequest.h

```diff
@@ -0,0 +1,94 @@
+#import <Foundation/Foundation.h>
+
+@class OIDServiceConfiguration;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface OIDLogoutRequest : NSObject <NSCopying, NSSecureCoding> {
+    OIDServiceConfiguration *_configuration;
+    NSURL *_postLogoutRedirectURL;
+    NSString *_idTokenHint;
+    NSString *_state;
+    NSDictionary<NSString *, NSString *> *_additionalParameters;
+}
+
+/*! @brief The service's configuration.
+ @remarks This configuration specifies how to connect to a particular OAuth provider.
+ Configurations may be created manually, or via an OpenID Connect Discovery Document.
+ */
+@property(nonatomic, readonly) OIDServiceConfiguration *configuration;
+
+/*! @brief The client's redirect URI.
+ @remarks post_logout_redirect_uri
```

Comment:
Indentation is off, here and other comment documentation blocks, please follow style of the rest of the project.